rita_bin/src/extender.rs

```diff
@@ -0,0 +1,75 @@
+//! This is the main source file for the Rita extender binary
+
+#![warn(clippy::all)]
+#![allow(clippy::pedantic)]
+#![forbid(unsafe_code)]
+
+#[cfg(feature = "jemalloc")]
+use jemallocator::Jemalloc;
+#[cfg(feature = "jemalloc")]
+#[global_allocator]
+static GLOBAL: Jemalloc = Jemalloc;
+
+#[macro_use]
+extern crate log;
+
+use rita_client::get_device_mac;
+use rita_common::logging::enable_remote_logging;
+use rita_common::utils::env_vars_contains;
+use rita_extender::extender_checkin;
+use rita_extender::get_checkin_message;
+use rita_extender::start_rita_extender_loop;
+use rita_extender::DEFAULT_UPSTREAM_ENDPOINT;
+
+#[actix_rt::main]
+async fn main() {
```

Comment:
the actix_rt main is good, but you can't use the wg key of the router since the extender won't have any wireguard keys (or anything to do with wireguard actually).

Why do we need to hardcode? we should get the mac address from the system on the fly.

Reply:
I meant using the wg key of the main router
extender queries endpoint -> main router return its wgkey -> we use it to log so that we know which router an extender belongs to in greylog